### PR TITLE
translation(pt-br): Translate half of the features section

### PR DESCRIPTION
**Pull request by felipesuri**
Continuing Portuguese Brazil translation of the docs. (#1987)

- [x] Translate half of the features section.

## Translated pages:

- [x] 1.rendering-modes
- [x] 2.deployment-targets
- [x] 3.file-system-routing
- [x] 4.data-fetching

### CODE REVIEW

File: content/pt-br/docs/3.features/1.rendering-modes.md
Comment (smarroufin) [inline content/pt-br/docs/3.features/1.rendering-modes.md:23]:
```suggestion
Você não precisa adicionar `ssr: true` em sua configuração do Nuxt para habilitar a renderização do lado do servidor, pois ela é habilitada por padrão.
```

File: content/pt-br/docs/3.features/4.data-fetching.md
Comment (smarroufin) [inline content/pt-br/docs/3.features/4.data-fetching.md:147]:
```suggestion
Você pode utilizar a diretiva `keep-alive` nos componentes `<nuxt />` e `<nuxt-child />` para salvar as chamadas ao `fetch` em páginas que você visitou.
```

**Review by smarroufin — CHANGES_REQUESTED** [review]
Nice work, thanks! 💚 

LGTM except those 2 typos.
Also waiting for @ChristopheCVB approval.

File: content/pt-br/docs/2.concepts/1.views.md
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/1.views.md:48]: You sure this is the right link?

File: content/pt-br/docs/3.features/2.deployment-targets.md
Comment (ChristopheCVB) [inline content/pt-br/docs/3.features/2.deployment-targets.md:26]:
```suggestion
- Remove `req` e `res` do `context`
```

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Just some typos to fix

**Comment by felipesuri** [comment]
Done @ChristopheCVB and @smarroufin.

**Review by ChristopheCVB — APPROVED** [review]
👍 Terrific work, again!

**Review by smarroufin — APPROVED** [review]
Thanks again! 💚

**Pull request merged**
